cirq-google/cirq_google/calibration/phased_fsim.py

```diff
@@ -641,6 +654,10 @@ def _json_dict_(self) -> Dict[str, Any]:
         }
 
 
+def _correlated_from_readout_tolerance(readout_tolerance: float) -> float:
+    return max(0.0, min(1.0, 7 / 6 * readout_tolerance - 1 / 6))
```

Comment:
provide a short explanation for this formula?

Reply:
Done, I've also commented on the keys in the args dict themselves.